packages/host/app/services/operator-mode-state-service.ts

```diff
@@ -11,6 +11,7 @@ import { service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 import { getOwner } from '@ember/application';
 import { scheduleOnce } from '@ember/runloop';
+import { Card } from 'https://cardstack.com/base/card-api';
```

Comment:
please import only the type here. we don't actually need the `Card` implementation just its types. so importing for just the type will result in this import being stripped out when transpiled to js.